### PR TITLE
add test for click santehnika

### PR DESCRIPTION
**Pull request by sauronx2**

### CODE REVIEW

File: src/test/java/test/TestRozetkaURL.java [hunk 1]
```diff
@@ -25,4 +28,10 @@ public void testSearchResult(){
         RozetkaResultSeach rozetkaResultSeach = new RozetkaResultSeach(driver);
         rozetkaResultSeach.searchField().sendKeys("монитор\n");
     }
+    @Test
+    public void getSix(){
+        ElementRozetka elementRozetka = new ElementRozetka(driver);
+        elementRozetka.goToRozetkaUa();
+        elementRozetka.getListSant().get(6).click();
+    }
```
Comment (OleksandrMoroz) [inline src/test/java/test/TestRozetkaURL.java:36]: Add Assert nooooooB

Comment (sauronx2) [reply src/test/java/test/TestRozetkaURL.java:37]: fck you i added assert url

Comment (OleksandrMoroz) [reply src/test/java/test/TestRozetkaURL.java:37]: fuk YOU add space bitwine test element

Comment (sauronx2) [reply src/test/java/test/TestRozetkaURL.java:38]: you stupid peace of shit shut the fuck up retard

just "\n" betwiane test and assert!!!

Comment (OleksandrMoroz) [reply src/test/java/test/TestRozetkaURL.java:38]: Fuck YOU but you NOT add "\n" after list 35-36

Comment (OleksandrMoroz) [reply src/test/java/test/TestRozetkaURL.java:38]: Good

**Pull request merged**